src/App.tsx

```diff
@@ -0,0 +1,187 @@
+import {
+  AppBar,
+  Alert,
+  Box,
+  Button,
+  Container,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  Toolbar,
+  Typography,
+} from "@mui/material";
+import SendIcon from "@mui/icons-material/Send";
+import React from "react";
+import CodeMirror from "@uiw/react-codemirror";
+import { oneDark } from "@codemirror/theme-one-dark";
+import init, { program } from "miden-wasm";
+import "./App.css";
+
+async function getExample(example: string[]) {
+    const inputs = fetch(`https://raw.githubusercontent.com/0xPolygonMiden/examples/main/examples/${example}.inputs`)
+    const masm = fetch(`https://raw.githubusercontent.com/0xPolygonMiden/examples/main/examples/${example}.masm`)
+    return [(await inputs).text(), (await masm).text()];
+  }
+
+function App() {
+
+  const [inputs, setInputs] = React.useState(
+    `{
+      "stack_init": ["0"],
+      "advice_tape": ["0"]
+}`
+  );
+
+  var exampleCode =`begin
+  push.1
+  push.2
+  add
+end`
+
+  const [code, setCode] = React.useState(exampleCode);
+
+  const [output, setOutput] = React.useState({
+    init: false,
+    success: true,
+    text: "",
+  });
```

Comment:
let's add a constant for empty output here so we can reset it when the example is changed:

```

  const emptyOutput = {
    init: false,
    success: true,
    text: "",
  };
  const [output, setOutput] = React.useState(emptyOutput);
```